Dockerfile

```diff
@@ -17,5 +17,5 @@ RUN dnf -y install \
 
 # Install eslint
 RUN npm install -g eslint && \
-	npm cache rm
+	npm cache clean --force
```

Comment:
Ah, now I get it! I was going to say `--force` is an undocumented flag for `cache`, and in a sense it is... but `npm` commands take configuration options anywhere, and `force` is a configuration flag. So this command's equivalent to `npm config set force=true && npm cache clean && npm config set force=false`. Frickin' NPM.

Works for me!

Reply:
I'd merge this, but I don't have access here. :grin:

Reply:
These GitHub permissions got complicated all of a sudden. Added you as an admin here, too!

Reply:
And how! I just tried to go and make the repo's newly-deployed package public, but... #ComputerSaysNo!

![Screenshot from 2022-03-21 21-26-57](https://user-images.githubusercontent.com/538020/159389147-217be96e-3c86-4d32-bec2-dabd5c44d12b.png)

Now I feel all dis-<i>org</i>anized!

Reply:
(It's a one-time deal, AIUI, so I don't really need or care about permission to do it **myself**... as long as someone does it, I'm happy.)

Reply:
Seriously! I guess there's another permission for setting permission permissions :laughing: 

I marked it public and explicity set `GSConnect/gnome-shell-extension-gsconnect` as having read access. That should clear up any permission issues for GitHub Actions.

Reply:
Indeed... though to actually make use of that access, the workflow on the g-s-e-gsconnect repo side would have to log in to the Container Registry with its `GITHUB_TOKEN` — something it currently isn't doing (because there's no need).

But next week or whenever, when something breaks and public access stops working, we can always just add the credentials configs.